lib/rubocop/runner.rb

```diff
@@ -27,7 +27,11 @@ def initialize(options, config_store)
 
     def run(paths)
       target_files = find_target_files(paths)
-      inspect_files(target_files)
+      if @options[:list_target_files]
+        puts target_files.map { |file| PathUtil.relative_path(file) }
```

Comment:
at first I had it as another method. I violated the rubocop max class length, so I tried to compact it a little bit to minimize that... I think the one line form is still fairly expressive but I'm happy to re-extract it